packages/trader/src/Modules/Contract/Containers/contract-replay.jsx

```diff
@@ -257,6 +257,7 @@ const Chart = props => {
             chartType={props.chart_type}
             endEpoch={props.end_epoch}
             margin={props.margin || null}
+            handlePOCResponse={props.is_accumulator_contract ? props.refToAddTick : undefined}
```

Comment:
Shouldn't we add this line to the trade.jsx too?

Reply:
@balakrishna-deriv no, as per requirements, we want to do it only in Contract Details, and only for Accumulator.🙂